### PR TITLE
Add a way to use HAOS in Windows on ARM.

### PR DESCRIPTION
**Pull request by ms300**
Add a way to use HAOS in Windows on ARM.
relevant issue: https://github.com/home-assistant/operating-system/issues/2256

## Proposed change

Add a way to use HAOS in Windows on ARM.
relevant issue: https://github.com/home-assistant/operating-system/issues/2256

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/operating-system/pull/2262
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -300,6 +300,10 @@ _All these can be extended if your usage calls for more resources._
   content: |
     <div class='note warning'>
         Hyper-V does not have USB support
+       
+        Hyper-V in Windows on ARM is supported from Home Assistant OS 10.0 generic aarch64. Before you create a virtual machine, an image conversion should be done.
```
Comment (frenck) [inline source/_includes/installation/operating_system.md:304]: I think this means it is not supported, as 10.0 doesn't exist as of yet.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/installation/operating_system.md (hunk 1 above)
Comment (ms300) [reply source/_includes/installation/operating_system.md:304]: @frenck It is verified on HAOS 10.0 20221210 dev version. So I use the `next` branch for the upcoming version of Home Assistant. Or maybe this merge should be pending until 10.0 released?

Comment (frenck) [reply source/_includes/installation/operating_system.md:304]: > It is verified on HAOS 10.0 20221210 dev version

Sure, but that isn't a public stable release, and, thus should not be listed in our public stable documentation.

../Frenck

**Comment by frenck** [comment]
I will close and friendly decline this PR because of the above-provided reasoning.

../Frenck

**Pull request closed without merging**